finalfrontier-utils/src/bin/ff-deps.rs

```diff
@@ -91,20 +95,25 @@ fn do_work<P, R>(
     let f = File::open(corpus_path.into()).or_exit("Cannot open corpus for reading", 1);
     let (data, start) =
         thread_data_conllx(&f, thread, n_threads).or_exit("Could not get thread-specific data", 1);
+    let projectivizer = if projectivize {
+        Some(HeadProjectivizer::new())
+    } else {
+        None
+    };
 
-    let mut sentences = Reader::new(BufReader::new(&data[start..]));
+    let mut sentences = SentenceIter::new(BufReader::new(&data[start..]), projectivizer);
```

Comment:
Maybe clone the projectivizer here, so that you do not have to create a new one below?

Reply:
Ideally the projectivizer implements both `Copy` and `Clone` since it's just a dummy struct. Unfortunately it doesn't implement those traits, so we have to construct a new one and have the if/else here

Reply:
It should. I have just uploaded conllx 0.11.1 with a Copy + Clone HeadProjectivizer.